app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -51,12 +52,11 @@ def get_feature(self, obj):
                     fv for fv in feature_values if fv.feature_config == feature_config
                 ][0]
                 feature_enabled = branch_feature_value.enabled
-                try:
-                    feature_value = json.loads(branch_feature_value.value)
-                except json.JSONDecodeError:
+
+                with contextlib.suppress(json.JSONDecodeError):
```

Comment:
interesting

Reply:
I know I've never seen this pattern before either.  It's very nice.

Reply:
What the heck this is amazing!